app/src/main/java/ch/epfl/sweng/eventmanager/ui/eventShowcase/fragments/schedule/ScheduleParentFragment.java

```diff
@@ -48,23 +57,83 @@ public void onCreate(Bundle savedInstanceState) {
         setRetainInstance(true);
     }
 
+    @Override
+    public void onResume() {
+        super.onResume();
+
+        Log.i(TAG, "Resume " + scheduleViewModel);
```

Comment:
Please move this after the null check: "resume null" isn't useful! You should also replace "resume" by "Resuming model from memory: " or similar.

Reply:
Fixed !